src/github.com/getlantern/flashlight/genconfig/genconfig.go

```diff
@@ -69,6 +75,7 @@ func main() {
 	runtime.GOMAXPROCS(numcores)
 
 	loadDomains()
+	loadWhitelists()
```

Comment:
The names here could stand to be cleaned up.  Blacklist and whitelist sound like they're related, but they're not.  Blacklist is a list of domains which we don't want to use for masquerading, whereas whitelist is a list of proxied sites.  In fact, though it'll be a little bit of a pain, I'm wondering if we shouldn't refer to it as proxiedSitesList throughout the code?  Whitelist is a pretty generic term that can apply to a bunch of different things.